.travis.yml

```diff
@@ -1,12 +1,25 @@
 language: node_js
 node_js:
-- '8'
+  - "8.11.2"
+
 cache: yarn
+
+install:
+  - yarn global add lerna
+  - lerna bootstrap
+
+matrix:
+  include:
+    env: PROJECT=packages/app
+
 script:
-- yarn lint 
-- yarn test
+  - cd $PROJECT
+  - yarn test
+  - yarn lint
```

Comment:
Reading the lerna docs – looks like this can be done without the matrix thing by running
 
    lerna run test
    lerna run lint

Reply:
that's a good shout, I'll see how this works for package that won't need to be deployed though, like our resuable components package